varnish2_1_5.rb

```diff
@@ -2,14 +2,9 @@
 
 class Varnish2_1_5 < Formula
   url 'http://repo.varnish-cache.org/source/varnish-2.1.5.tar.gz'
-  head 'http://repo.varnish-cache.org/source/varnish-3.0.0-beta1.tar.gz'
```

Comment:
This formula is for a specific version of Varnish for vendor compatibility reasons, thus the hardcoding of the version number in the formula name. HEAD is also a full version ahead of this script - script will not be used to install new versions, so removed the option.